docs/pull-requests.md

```diff
@@ -139,12 +139,13 @@ If you sense that a conversation between you and the author is proceeding in an
 
 If you need to take over and merge someone else's branch, let them know so they aren't surprised.
 
-### Rebase sparingly
+### Rebase with caution
 
-Once you've created a pull request, avoid amending, rebasing, or otherwise altering the history of the branch, but push up each new change as a new commit instead. This has two advantages:
+Once you've created a pull request, avoid amending, rebasing, or otherwise altering the history of the branch, but push up each new change as a new commit instead. This has a few advantages:
 
-1. It keeps reviewers in the loop as changes are made. Keep in mind that they won't check your pull request as often as you will. If you've made a bunch of updates in response to feedback and now need a second round of reviews, then your reviewers can look over the list of most recent commits to catch up.
-2. It creates a smooth workflow with collaborators. If you're working with someone else but you change the history of the branch, they may have to now blow away their version of the branch and re-fetch it. This creates friction, causes frustration, and wastes time.
+1. It aids reviewers in following your pull request over time. Keep in mind that they aren't as focused on your pull request as you are; they might look over your code, write some comments, leave, and then come back at a later time. The Conversation view of a pull request is an essential tool for reviewers to understand what happened while they were away. Usually, the timeline view preserves events in the same order in which they occurred. But rebasing the branch changes that order with regard to commits: specifically, if they were commingled with conversations before, now they are listed at the very end of the timeline. This can be extremely jarring.
+2. It ensures that conversations on a pull request stay as they are. Rebasing is not only confusing to reviewers because it jumbles the history of the pull request, but it can also be confusing to GitHub. If a commit connected to a conversation changes enough, GitHub may think that the conversation is outdated and mark it as such. This is misleading and may cause the author of the pull request to start ignoring the conversation.
+3. It creates a smooth workflow with collaborators. If you're working with someone else but you change the history of the branch, they may have to blow away their version of the branch and re-fetch it. This creates friction, causes frustration, and wastes time.
```

Comment:
nit: style
- digestible subject line + bullet points
- neutral tone, avoid "you" statements

```suggestion
1. It preserves the order of timeline activity in the Conversation view, helping reviewers follow pull requests over time. 
- Keep in mind that reviewers aren't as focused on pull requests as their authors; they might review the code in parcels, leaving and returning many times in between comments. The Conversation view of a pull request is an essential tool for reviewers to understand what happened while they were away. 
- Rebasing a branch changes the commit order in the Conversation view: specifically, commits that were commingled with conversations before will now be listed at the tail end of the timeline. This makes it very confusing for reviewers to pick up where they left off.
2. It ensures that active conversations aren't marked as outdated. 
- Rebasing confuses not only reviewers by jumbling the history of a pull request — it can also confuse GitHub. 
- If a commit connected to a conversation changes enough, GitHub may mark it as outdated. This is misleading and can cause the author of the pull request to inadvertently ignore new comments in ongoing conversations.
3. It creates a smoother workflow for collaborators. 
- If the history of a PR branch is rewritten by a rebase, collaborators may have to reset and re-fetch their local branch. 
- This creates unnecessary friction and wastes time.
```

Reply:
I've tended to use bullet points for lists, like "here are some reasons why this is useful" rather than "here are some items on a slide". But, I can see how useful it is to break up the text visually. Whether to use "you" seems like a separate discussion, but I see how removing it is better.

I've taken your suggestion as inspiration and made these changes here: [18099b7](https://github.com/MetaMask/contributor-docs/pull/60/commits/18099b72f1bbb36ad27d77efda127121e45937b4). Are they any better?

Reply:
This looks good! I'm happy with this edit.